apps/passport-server/src/services/rateLimitService.ts

```diff
@@ -59,33 +54,35 @@ export class RateLimitService {
     actionType: RateLimitedActionType,
     actionId: string
   ): Promise<boolean> {
-    return traced(
-      "RateLimitService",
-      "requestRateLimitedAction",
-      async (span) => {
-        span?.setAttribute("actionType", actionType);
-        span?.setAttribute("actionId", actionId);
+    return true;
```

Comment:
this change is not intended to make it into the `main` branch. i needed to make it in order to be able to test this PR properly. cc @robknight - what do you think the best way to modify/disable the rate limit for local development would be?

Reply:
`UPDATE rate_limit_types SET hourly_limit = 1000000`

Reply:
I will look at adding a setting/environment variable for turning off rate limiting though